spec/helpers/appropriate_body_helper_spec.rb

```diff
@@ -97,19 +105,22 @@
 
   describe '#trs_alerts_text' do
     context 'when alerts are present' do
-      it 'returns yes' do
-        expect(trs_alerts_text(true)).to include('Yes')
-      end
+      it 'displays yes with a link to teacher record service' do
+        result = trs_alerts_text(true)
 
-      it 'also returns a sentence about getting more info' do
-        expect(trs_alerts_text(true)).to include('Use the', 'to get more information')
-        expect(trs_alerts_text(true)).to include('https://www.gov.uk/guidance/check-a-teachers-record')
+        expect(result).to have_text('Yes')
```

Comment:
I'd favour writing the specs this way, there is overlap, but I think it communicates things more clearly.

```ruby
expect(result).to have_text("Use the Check a teacher's record service to get more information")
expect(result).to have_link("Check a teacher's record service", href: "https://www.gov.uk/guidance/check-a-teachers-record")
```